egs/wsj/s5/steps/dur_model/make_nnet_config.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
you can take a look at steps/nnet3/*/make_configs.py scripts for an example of a generic config generator. You can make use of steps/nnet3/components.py